Update muon interface docs (#25781)
mantidproject/mantid: docs/source/interfaces/muon_phase_table_tab.rst

.. _muon_phase_table_tab-ref:

Phase Table Tab
---------------

.. image::  ../images/muon_interface_tab_phase_table.png
   :align: right
   :height: 500px

Phase Table Calculator
^^^^^^^^^^^^^^^^^^^^^^

The phase table calculator provides an interface to generate a table of detector phases and asymmetries.

**Workspace** This selects the workspace for which the phase table will be calculated.

**Forward group** This selects the forward detector group.

**Backward group** This selects the backward detector group.

**First good data** This selects the first good data value used, this is set separately to the value on the home tab.

**Last good data** This selects the last good data value used, this is set separately to the value on the home tab.

> **Output fit information** Check this box to generate a workspace of fit information for your phase table.

Author: ```suggestion
**Output fit information** If Checked this will generate a workspace of fit information for your phase table.
```
